src/verilator.ts

```diff
@@ -177,16 +177,26 @@ export class VerilatorDiagnostics {
                     line = line.replace(/\s+/g, ' ').trim();
 
                     let terms = this._splitTerms(line);
-                    let severity = this._getSeverity(terms[0]);
-                    let message = terms.slice(2).join(' ')
-                    let lineNum = parseInt(terms[1].trim()) - 1;
+                    if (terms == null) {
+                        ConnectionLogger.warn('Unexpected Verilator output format: ' + line);
```

Comment:
Trying this on older versions of verilator (e.g. 3.9), this causes extraneous reporting. So best not have this warning. Just the return should be enough.